src/OrchardCore/OrchardCore.ContentManagement.Abstractions/Metadata/Settings/ContentPartFieldSettingsExtensions.cs

```diff
@@ -23,5 +23,10 @@ public static ContentPartFieldDefinitionBuilder WithDisplayMode(this ContentPart
         {
             return builder.MergeSettings<ContentPartFieldSettings>(x => x.DisplayMode = displayMode);
         }
+
+        public static ContentPartFieldDefinitionBuilder WithPosition(this ContentPartFieldDefinitionBuilder builder, string position)
```

Comment:
We need to check `position` at least if it has a value, also we can use inline lambda function

Reply:
I don't understand @hishamco ?
Why would it matter if it has a value or not. 
See the rest of the code around this area, it is consistent.

Reply:
If the position could be null or empty which I don't think, ignore my comment